tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -1915,9 +1916,10 @@ public function quickCleanUpFinancialEntities() {
       'civicrm_price_field',
     ];
     $this->quickCleanup($tablesToTruncate);
+    $this->disableTaxAndInvoicing();
```

Comment:
Actually this line was removed for no reason in the end - doesn't it look prettier here though?